src/index.js

```diff
@@ -99,7 +99,7 @@ const ApolloClientPlugin = createPlugin({
 
         return forward(operation);
       });
-      const links = getApolloLinks([authMiddleware, connectionLink]);
+      const links = getApolloLinks(ctx)([authMiddleware, connectionLink]);
```

Comment:
Nice, but I'm not sure we need to have two function invocations. Why not pass an array of links, as well as context in?

Reply:
@KevinGrandon Because, i think the client interface will be more clean (maybe?)
```
export default (ctx) => ([authMiddleware, connectionLink]) => {
}
```